java/core/src/java/org/apache/orc/impl/RecordReaderUtils.java

```diff
@@ -545,7 +545,7 @@ public int hashCode() {
 
     private long currentGeneration = 0;
 
-    private final TreeMap<Key, ByteBuffer> getBufferTree(boolean direct) {
```

Comment:
Let's revert this change.

Reply:
Any reason for revert or you prefer this also goes into a separate PR?

Reply:
Merged as part of ORC-754